.tests/cypress/integration/mini-cart/remove-0qty-from-cart-configurable.js

```diff
@@ -1,87 +1,83 @@
+        'permission-profiling',
+        'PHPSESSID',
+        'form_key',
+      ]
     })
+  })
 
-    it('Visits product', () => {
-        cy.visit('https://alpaca-ce-solr-demo.snowdog.pro/configurable-product')
-        cy.get('.breadcrumbs__list').should('be.visible')
-    })
-
-    it('Check visiblility of content', () => {
-        cy.get('.product-view__main-details').should('be.visible')
-        cy.get('[data-ui-id=page-title-wrapper]').should('be.visible')
-        cy.get('.product-view__information').should('be.visible')
-        cy.get('.product-view__sku').should('be.visible')
-        cy.get('#product-price-2048').should('be.visible')
-        cy.get('.quantity-update').should('be.visible')
-        cy.get('#product-addtocart-button').should('be.visible')
-        cy.get('.product-view__extra-actions').should('be.visible')
-        cy.get('[data-testid=product-tab__title]').contains('Details').should('be.visible')
-        cy.get('[data-testid=product-tab__title]').contains('Reviews').should('be.visible')
+  it('Visits product', () => {
+    cy.visit('https://alpaca-ce-solr-demo.snowdog.pro/configurable-product')
+    cy.get('.breadcrumbs__list').should('be.visible')
+  })
 
-    })
+  it('Check visiblility of content', () => {
+    cy.get('.product-view__main-details').should('be.visible')
+    cy.get('[data-ui-id=page-title-wrapper]').should('be.visible')
+    cy.get('.product-view__information').should('be.visible')
+    cy.get('.product-view__sku').should('be.visible')
+    cy.get('#product-price-2048').should('be.visible')
+    cy.get('.quantity-update').should('be.visible')
+    cy.get('#product-addtocart-button').should('be.visible')
+    cy.get('.product-view__extra-actions').should('be.visible')
+    cy.get('[data-testid=product-tab__title]').contains('Details').should('be.visible')
+    cy.get('[data-testid=product-tab__title]').contains('Reviews').should('be.visible')
+  })
 
-    it('Checks configurable options', () => {
+  it('Checks configurable options', () => {
     cy.get('.swatch-opt').should('be.visible').find('.size')
-        cy.get('.size').find('.swatch__option')
-            .then((elements) => {
-                cy.log(elements)
-                elements[Math.floor(Math.random() * elements.length)]
-                    .click(elements)
-            })        
-        cy.get('.color').find('.swatch__option')
-            .then((elements) => {
-                cy.log(elements)
-                elements[Math.floor(Math.random() * elements.length)]
-                    .click(elements)
-            })
-        cy.get('#product-addtocart-button').first()
-            .click()
-            cy.server()
-            cy.route('/customer/section/load/?sections=cart*').as('addToCart')
-            cy.wait('@addToCart')
-        })
+    cy.get('.size').find('.swatch__option')
+      .then((elements) => {
+        cy.log(elements)
+        elements[Math.floor(Math.random() * elements.length)]
```

Comment:
Please write a function and then cal it here instead. Please see cypress best practise and previous comments to proceed.